packages/manager/src/features/CloudPulse/Alerts/AlertsResources/AlertsResources.tsx

```diff
@@ -131,17 +131,15 @@ export const AlertResources = React.memo((props: AlertResourcesProp) => {
 
   const supportedRegionIds = getSupportedRegionIds(regions, serviceType);
   const xFilterToBeApplied: Filter | undefined = React.useMemo(() => {
-    if (serviceType === 'firewall') {
+    if (serviceType === 'firewall' || supportedRegionIds?.length === 0) {
```

Comment:
```suggestion
    if (serviceType === 'firewall' || !supportedRegionIds?.length) {
```

when supportedRegionIds are undefined then this statement will become false